cmd/traffic_ctl/traffic_ctl.cc

```diff
@@ -30,13 +30,27 @@
 
 AppVersionInfo CtrlVersionInfo;
 
-const char *CtrlMgmtRecord::name() const { return this->ele->rec_name; }
+const char *
```

Comment:
This is what I see when I look at the current master:
``
const char *
CtrlMgmtRecord::name() const
{
  return this->ele->rec_name;
}

Reply:
This was recently fixed... checking...

Reply:
This is a duplicate of #3271 I'll close it.